.github/workflows/main.yml

```diff
@@ -126,3 +126,23 @@ jobs:
         SSH_CONFIG: ${{ secrets[env.SSH_CONFIG_SECRET_NAME] }}
         SSH_KEY: ${{ secrets[env.SSH_KEY_SECRET_NAME] }}
       run: ./.github/scripts/deploy.sh
+
+    - name: Jira Login
+      if: env.TIER == 'PROD'
+      uses: atlassian/gajira-login@master
+      env:
+        JIRA_BASE_URL: ${{ secrets.JIRA_BASE_URL }}
+        JIRA_USER_EMAIL: ${{ secrets.JIRA_USER_EMAIL }}
+        JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
+
+    - name: Create Jira Transition List
+      if: env.TIER == 'PROD'
+      run: curl https://terraware.github.io/terraware-web/unreleased.log | grep -E 'SW-[0-9]+' -o | sort -u > ./docs/jiralist.txt
```

Comment:
Should this be looking at terraware-server instead of terraware-web?

Also, this would be easier to read if it were split into multiple lines or extracted to a script.

Reply:
Good catch, bad copy/paste job